app/views/courses/new.html.erb

```diff
@@ -1,5 +1,3 @@
-<% @title = "Create New Course" %>
```

Comment:
Keep the title on the page itself rather than the partial (since partials could be used in multiple places)